scripts/tests.sh

```diff
@@ -1,42 +1,45 @@
+#!/bin/bash
+
 # create tests directory and test case file
 mkdir tests
 touch tests/test_main.py
 
 # add continuity test
-echo '
+cat << EOF > tests/test_main.py
 import requests 
 import argparse
+import uuid
 
 from basebot import TheMessage, MessageWrapper, MessageContents
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--port', type=int, help='localhost:{port}', default=8000)
+parser.add_argument("--port", type=int, help="localhost:{port}", default=8000)
 args = parser.parse_args()
 
-bot_name = input('BotName: ')
-local_uuid = 'test1f3a-97ee-4e7e-8242-7b1d202c0fb5'
-url = f'http://localhost:{args.port}/bots/{bot_name}'
-resp = requests.get(url+'/about').json()
-bot_id = resp['bot_id']
+bot_name = input("Which bot are you trying to test?\n\nBotName: ")
+local_uuid = str(uuid.uuid4())
```

Comment:
Having a static uuid makes the db not bloat, and you can easily clear all of the test messages. As well as use the same id in other test scripts.